presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -246,6 +247,11 @@ public HiveSessionProperties(HiveClientConfig hiveClientConfig, OrcFileWriterCon
                         STATISTICS_ENABLED,
                         "Experimental: Expose table statistics",
                         hiveClientConfig.isTableStatisticsEnabled(),
+                        false),
+                integerSessionProperty(
+                        PARTITION_STATISTICS_SAMPLE_SIZE,
```

Comment:
This should be ensured it's non-negative (actually: positive, just as `hiveClientConfig.getPartitionStatisticsSampleSize()` must be).